src/overlays/actors/ovl_En_Ig/z_en_ig.c

```diff
@@ -20,13 +20,44 @@ void EnIg_Draw(Actor* thisx, GlobalContext* globalCtx);
-    0x1037110F, 0x0B0E1032, 0x1037060E, 0x0F2D0F32, 0x050E0F19, 0x0F2D0A0A, 0x00615A02, 0x0F2D0F32,
-    0x4E020F32, 0x100A4202, 0x100A101E, 0x36004B10, 0x1902101E, 0x10320D02, 0x10321037, 0x01050E10,
-    0x32103708, 0x0E101E10, 0x320D0210, 0x1E10320D, 0x02103210, 0x3701050E, 0x10321037, 0x090E101E,
-    0x10320E0E, 0x100A101E, 0x030E0F32, 0x100A0C0E, 0x0F2D0F32, 0x07050500,
+static u8 D_80BF3260[] = {
+    /* 0x0000 */ SCHEDULE_CMD_CHECK_DAY_L(1, 0x00B6 - 0x0005),
+    /* 0x0005 */ SCHEDULE_CMD_CHECK_SCENE_S(SCENE_TOWN, 0x0057 - 0x0009),
+    /* 0x0009 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(15, 25, 15, 45, 0x0051 - 0x000F),
+    /* 0x000F */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(15, 45, 15, 50, 0x004B - 0x0015),
+    /* 0x0015 */ SCHEDULE_CMD_CHECK_FLAG_S(0x4B, 0x10, 0x001A - 0x0019),
+    /* 0x0019 */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x001A */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(16, 50, 16, 55, 0x0045 - 0x0020),
+    /* 0x0020 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(16, 55, 17, 15, 0x003F - 0x0026),
+    /* 0x0026 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(17, 15, 18, 0, 0x0039 - 0x002C),
+    /* 0x002C */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(18, 0, 6, 0, 0x0033 - 0x0032),
+    /* 0x0032 */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x0033 */ SCHEDULE_CMD_RET_TIME(18, 0, 6, 0, 4),
+    /* 0x0039 */ SCHEDULE_CMD_RET_TIME(17, 15, 6, 0, 2),
+    /* 0x003F */ SCHEDULE_CMD_RET_TIME(16, 55, 17, 15, 11),
+    /* 0x0045 */ SCHEDULE_CMD_RET_TIME(16, 50, 16, 55, 6),
+    /* 0x004B */ SCHEDULE_CMD_RET_TIME(15, 45, 15, 50, 5),
+    /* 0x0051 */ SCHEDULE_CMD_RET_TIME(15, 25, 15, 45, 10),
+    /* 0x0057 */ SCHEDULE_CMD_CHECK_SCENE_S(SCENE_YADOYA, 0x00B5 - 0x005B),
+    /* 0x005B */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(15, 45, 15, 50, 0x00AF - 0x0061),
+    /* 0x0061 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(15, 50, 16, 10, 0x00A9 - 0x0067),
+    /* 0x0067 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(16, 10, 16, 30, 0x00A3 - 0x006D),
+    /* 0x006D */ SCHEDULE_CMD_CHECK_FLAG_S(0x4B, 0x10, 0x008A - 0x0071),
+    /* 0x0071 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(16, 30, 16, 50, 0x0084 - 0x0077),
+    /* 0x0077 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(16, 50, 16, 55, 0x007E - 0x007D),
+    /* 0x007D */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x007E */ SCHEDULE_CMD_RET_TIME(16, 50, 16, 55, 8),
+    /* 0x0084 */ SCHEDULE_CMD_RET_TIME(16, 30, 16, 50, 13),
+    /* 0x008A */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(16, 30, 16, 50, 0x009D - 0x0090),
+    /* 0x0090 */ SCHEDULE_CMD_CHECK_TIME_RANGE_S(16, 50, 16, 55, 0x0097 - 0x0096),
+    /* 0x0096 */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x0097 */ SCHEDULE_CMD_RET_TIME(16, 50, 16, 55, 9),
+    /* 0x009D */ SCHEDULE_CMD_RET_TIME(16, 30, 16, 50, 14),
+    /* 0x00A3 */ SCHEDULE_CMD_RET_TIME(16, 10, 16, 30, 3),
+    /* 0x00A9 */ SCHEDULE_CMD_RET_TIME(15, 50, 16, 10, 12),
+    /* 0x00AF */ SCHEDULE_CMD_RET_TIME(15, 45, 15, 50, 7),
+    /* 0x00B5 */ SCHEDULE_CMD_RET_NONE(),
+    /* 0x00B6 */ SCHEDULE_CMD_RET_NONE(),
 };
```

Comment:
```suggestion
    /* 0xAF */ SCHEDULE_CMD_RET_TIME(15, 45, 15, 50, 7),
    /* 0xB5 */ SCHEDULE_CMD_RET_NONE(),
    /* 0xB6 */ SCHEDULE_CMD_RET_NONE(),
};
```
It looks like all of these command scripts could have some 0's cut out